ltclient/script/deploy.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+[ ! -f dist/app.bundle.js ] && npm run build
+
+cp -r dist/* ../ltserver/client
```

Comment:
It can be done in gulp.

But this is a very simple task. You can use gulp pipe for the `cp` command. But for the `npm run build`, with gulp you need to use `child_process.spawn`. It's more intuitive to use bash script.

I'm also a Bash fanboy :)